RHEL6_7/databases/postgresql/check

```diff
@@ -340,7 +340,7 @@ EOF
     # Even if user misconfigured the pg_hba.conf contents, most likely he did
     # not changed the 'postgres' role.  Thus we can assume he is Superuser (with
     # slight risk).
-    return $RESULT_PASS
+    return $RESULT_FAIL
```

Comment:
according to the message it seems that there should be set even high risk. As I read it properly, this module seems to need even separate pull-request for all changes.

Reply:
From the message it seems, that the severity of the issue depends on the 'postgres' role. With the default configuration, severity is low. I agree with module rewrite in separate pull request where, the logic to determine the 'postgres' role will be added.